tox.ini

```diff
@@ -68,21 +68,20 @@ deps =
     online: astroquery
     hypothesis: astroquery
     # Figure tests need a tightly controlled environment
-    figure-!devdeps: astropy==5.1.0
-    figure-!devdeps: matplotlib==3.5.2
+    figure-!devdeps: astropy==5.2.0
+    figure-!devdeps: matplotlib==3.6.2
```

Comment:
As a general testing strategy I think it's worth keeping these pinned to the lowest versions we support (of both astropy and Matplotlib), and then using `devdeps` as the most-recent version test. So I'm 👎 on this bump (unless it's time to bump these in our minimum dependencies too?)

Reply:
These versions have never been the oldest ones we support. We normally bump these when we encounter problems in the build like we had to pin numpy. This removes the pin below.